payment-manager/src/main/java/com/qpay/paymentmanager/mapper/WalletMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.qpay.paymentmanager.mapper;
+
+import com.qpay.paymentmanager.model.dto.WalletCreation;
+import com.qpay.paymentmanager.model.dto.WalletModification;
+import com.qpay.paymentmanager.model.entity.WalletEntity;
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+
+@Component
+public class WalletMapper {
+
+    public WalletEntity map(final WalletCreation walletCreation) {
+        return WalletEntity.builder()
+                .name(walletCreation.name())
+                .balance(new BigDecimal(0))
```

Comment:
instead of setting default value over here and in some places in your tests. You can add default value using builder for your walletEntity class https://www.baeldung.com/lombok-builder-default-value#builder-default